# InfantEukaryotes
Notebooks supporting the publication: 

*Strain-level overlap between infant and hospital fungal microbiomes revealed through de novo assembly of eukaryotic genomes from metagenomes*

# Genomes

All genomes reported in Table 1. are available in the genomes folder (files ending in `.fasta`) along with predicted proteins (files ending in `.faa`). More information on genomes in available in the file `genomeInfo.csv`

# Supplemental Tables and Figures 

Supplemental Figure S1. Effect of coverage on variant frequency determination as assessed through simulation of metagenomic reads.

Supplemental Figure S2. Raw variant frequency graphs used to determine ploidy of all de novo assembled genomes.

Supplemental Figure S3. Dendrogram of reference and de novo assembled C. parapsilosis genomes based on genome-wide average nucleotide identity (ANI). 

Supplemental Figure S4. Evaluation of relative abundance, genome length, and sequencing depth needed to achieve 8x sequencing coverage.

Supplemental Figure S5. Determination of aneuploidy for all de novo assembled genomes.

Supplemental Figure S6. Alternative determination of aneuploidy for genomes with high quality reference genomes.

Supplemental Table S1. Sequencing metadata for all infant and room metagenomic samples

Supplemental Table S4. Mapping-based abundance of eukaryote genomes in all samples

Supplemental Table S3. Metadata for statistical associations 

Supplemental Table S4. Statistical associations of samples containing eukaryotes with metadata.

# Jupyter Notebooks

Supplemental Table S3 = SampleMetadata_v1.csv

Supplemental Table S4 = MappingListGamma_genomeCoverage_v1.csv
